Dnn.CommunityForums/ForumSettings.ascx

```diff
@@ -214,13 +214,6 @@
                 <asp:Literal ID="ltrFullTextMessage" runat="server" />
             </span>
 		</div>
-		  <div class="dnnFormItem">
-			<dnn:label ID="lblMailQueue" runat="server" resourcekey="MailQueue" Suffix=":" />
```

Comment:
@johnhenley Do we also need to remove the translation from the resource file?

Reply:
done.